doxygen/dox/Glossary.dox

```diff
@@ -0,0 +1,565 @@
+    <P>A regular multidimensional pattern described by four vectors whose length
+    equals the rank of the pattern.</P>
+    <OL>
+      <LI><CODE>start</CODE> - the offset where the first block of the hyperslab begins</LI>
+      <LI><CODE>stride</CODE> - the offset between pattern blocks</LI>
+      <LI><CODE>count</CODE> - the number of blocks</LI>
+      <LI><CODE>block</CODE> - the extent of an individual pattern block</LI>
+    </OL>
+    <P>For example, the black squares on a (two-dimensional) chessboard with
+      origin at <CODE>(0,0)</CODE> can be represented as the union of two
+      hyperslabs representing the even <CODE>(0,2,4,6)</CODE> and
+      odd <CODE>(1,3,5,7)</CODE> rows, respectively.</P>
+    <IMG SRC="https://upload.wikimedia.org/wikipedia/commons/thumb/d/d7/Chessboard480.svg/176px-Chessboard480.svg.png"/>
+    <P>The hyperslab parameters for the even rows are: <CODE>start (0,0)</CODE>,
+      <CODE>stride (2,2)</CODE>, <CODE>count (4,4)</CODE>, <CODE>block
+        (1,1)</CODE>. Likewise the parameters for the odd rows are: <CODE>start
+        (1,1)</CODE>, <CODE>stride (2,2)</CODE>, <CODE>count
+        (4,4)</CODE>, <CODE>block (1,1)</CODE>.</P>
+</DD>
+</DL>
+
+\section GLS_I I
+
+<DL>
+  <DT>Identifier</DT>
+  <DD>An opaque, transient handle used by the HDF5 library to manipulate
+  in-memory representations of HDF5 items.</DD>
+</DL>
+
+\section GLS_L L
+
+<DL>
+  <DT>Library</DT>
+  <DD></DD>
+</DL>
+
+<DL>
+  <DT>Link</DT>
+  <DD>
+    <P>An named, uni-directional association between a source and a
+    destination. In HDF5, the source is always the HDF5 group that hosts the
+    link in its link collection.</P>
+    <P>There are several ways to specify a link's destination:</P>
+    <UL>
+      <LI>The address of an HDF5 object in the same HDF5 file; so-called hard
+      link.</LI>
+      <LI>A path name in the same or a different file; so-called soft or
+      external link.</LI>
+      <LI>User-defined</LI>
+    </UL>
+    <P>A link name can be any Unicode string that does not contain slashes
+      (<CODE>"/"</CODE>) or consists of a single dot character
+      (<CODE>"."</CODE>). A link name must be unique in a group's link
+      collection.</P>
+  </DD>
+</DL>
+
+\section GLS_M M
+
+<DL>
+  <DT>Metadata</DT>
+  <DD>Data that in a given context has a descriptive or documentation function
+  for other data. Typically, the metadata is small compared to the data it
+  describes.</DD>
+</DL>
+
+<DL>
+  <DT>Member</DT>
+  <DD>
+    <P>A link destination is sometimes referred to as a member of the link's
+    source (group). This way of speaking invites confusion: A destination (e.g.,
+    object) can be the destination of multiple links in the same (!) or
+    different groups. It would then be a "member" of a given group with
+    multiplicity greater than one and be a member of multiple groups.</P>
+    <P> It is the link that is a member of the group's link collection and not
+    link's destination.</P>
```

Comment:
Should this sentence be "It is the link that is a member of the group's link collection and not the link destination."?